packages/legacy/core/App/components/misc/IOSStatusBar.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Platform, View } from 'react-native'
+
+import { useTheme } from '../../contexts/theme'
+
+
+
+const IOSStatusBar: React.FC = ({ ...props }) => {
+    const { ColorPallet } = useTheme()
+    return (
+        <>
+            {Platform.OS === "ios" && <View style={{ height: 40, backgroundColor: ColorPallet.brand.primary }} {...props}></View>}
```

Comment:
Is there a calculation we can use instead of just 40 pixels? Thinking about different phones